wideq/washer.py

```diff
@@ -0,0 +1,123 @@
+import enum
+from typing import Optional
+
+from .client import Device
+
+
```

Comment:
I decided it would be better to not include other enum values that really aren't all that useful like `SpinSpeed` and `WaterTemp`.  When I compared the json my washer returned with that of another washer the enum values for the same key were different.  For example:

```python
{"WaterTemp": {
            "type": "Enum",
            "default": "0",
            "label": "@WM_MX_OPTION_WASH_TEMP_W",
            "option": {
                "0": "-",
                "1": "@WM_MX_OPTION_TEMP_TAP_COLD_W",
                "2": "@WM_MX_OPTION_TEMP_COLD_W",
                "3": "@WM_MX_OPTION_TEMP_ECO_WARM_W",
                "4": "@WM_MX_OPTION_TEMP_WARM_W",
                "5": "@WM_MX_OPTION_TEMP_WARM_W",
                "6": "@WM_MX_OPTION_TEMP_HOT_W",
                "7": "@WM_MX_OPTION_TEMP_EXTRA_HOT_W"
            }
        }
}
# compared to:
{"WaterTemp": {
            "default": "0",
            "option": {
                "0": "-",
                "2": "@WM_MX_OPTION_TEMP_COLD_F3L2CNV4W_WIFI_W",
                "3": "@WM_MX_OPTION_TEMP_SEMI_WARM_F3L2CNV4W_WIFI_W",
                "6": "@WM_MX_OPTION_TEMP_HOT_F3L2CNV4W_WIFI_W",
                "5": "@WM_MX_OPTION_TEMP_WARM_F3L2CNV4W_WIFI_W",
                "4": "@WM_MX_OPTION_TEMP_WARM_F3L2CNV4W_WIFI_W",
                "7": "@WM_MX_OPTION_TEMP_EXTRA_HOT_F3L2CNV4W_WIFI_W",
                "1": "@WM_MX_OPTION_TEMP_TAP_COLD_F3L2CNV4W_WIFI_W"
            }
```